### PR TITLE
Issue 42

### LINKED ISSUE #42: Page not found in Django 1.9 with simple action
```
class TaskAdmin(DjangoObjectActions, admin.ModelAdmin):
    def cancel(self, request, task):
        task.cancel()
    objectactions = 'cancel',
```
Go to admin page, press 'CANCEL' button and...
>task object with primary key '2/change/tools/cancel' does not exist.

Page URL is
>task/2/change/tools/cancel/change/

But if manually change URL to
>2/tools/cancel

All is OK.

I found what in https://docs.djangoproject.com/en/1.9/releases/1.9/#django-contrib-admin
```The URL of the admin change view has been changed (was at /admin/<app>/<model>/<pk>/ by default and is now at /admin/<app>/<model>/<pk>/change/). This should not affect your application unless you have hardcoded admin URLs. In that case, replace those links by reversing admin URLs instead. Note that the old URL still redirects to the new one for backwards compatibility, but it may be removed in a future version.```

### PR DESCRIPTION
**Pull request by Amareis**
Fixed #42
Also added support for Django 1.9.

### CODE REVIEW

**Comment by Amareis** [comment]
Ehm. So, we need to drop support Django 1.4 if want support 1.9. Old `url` tag behaviour deprecated, and `{%load url from future%}` removed from 1.9.

**Comment by crccheck** [comment]
totally ok with dropping Django 1.4. It breaks #38 too.

File: django_object_actions/templates/django_object_actions/change_form.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
 {% block object-tools-items %}
   {% for tool in objectactions %}
     <li class="objectaction-item" data-tool-name="{{ tool.name }}">
-      <a href='tools/{{ tool.name }}/' title="{{ tool.standard_attrs.title }}"
+      <a href='{%url tools_view_name pk=object_id tool=tool.name %}' title="{{ tool.standard_attrs.title }}"
```
Comment (crccheck) [inline django_object_actions/templates/django_object_actions/change_form.html]: whitespace consistency: I like keeping a single space between the braces so it's look like `{% url ... %}`

File: django_object_actions/utils.py [hunk 2]
```diff
@@ -14,17 +15,28 @@ class BaseDjangoObjectActions(object):
...
     objectactions = []
+    tools_view_name = None
 
-    def get_tool_urls(self):
+    def get_tool_urls(self, urls):
         """Gets the url patterns that route each tool to a special view."""
         tools = {}
+
+        end = '_change'
+        for u in urls:
```
Comment (crccheck) [inline django_object_actions/utils.py]: NAMING THINGS: I'll blame it on me being sleepy, but I'm having trouble reading this block of logic. Some quick rename `u` to `url`

```
is_change_page = url.name.endswith('_change')
if is_change_page:
   tools_view = url.name.replace('_change', '_tools')
```

the `for` ... `break` smells funny to me too but oh well, as long as it works. It makes me think there should be a test case covering if there is no `_change` url. Maybe that never happens.` ¯\_(ツ)_/¯`

Comment (Amareis) [inline django_object_actions/utils.py]: I tried to verify the absence of a url with "_change", but can not offer a test for this situation and CI fell on the test coverage. So I deleted this. If you have an idea of the test, I'll do it.

Comment (Amareis) [inline django_object_actions/utils.py]: I tried to verify the absence of a url with "_change", but can not offer a test for this situation and CI fell on the test coverage. So I deleted this. If you have an idea of the test, I'll do it.

**Pull request merged**